.github/workflows/examine-wasm-bindings.yml

```diff
@@ -77,10 +77,25 @@ jobs:
         working-directory: wallet/bindings/wasm/
         run: yarn build
 
+      - name: Start private tangle
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        uses: './.github/actions/private-tangle/setup'
+
+      - name: Run Yarn Test
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        working-directory: wallet/bindings/wasm/
+        run: yarn test
+
       - name: Run Yarn Test
+        if: ${{ !startsWith(matrix.os, 'ubuntu') }}
         working-directory: wallet/bindings/wasm/
+        # Only run test that doesn't require a node because we can't use the private tangle script on macos and windows
         run: yarn test
```

Comment:
Both ubuntu and !ubuntu are doing the same thing?

Reply:
@kwek20 skipped the local node test completely
Don't we want to have it still tested on ubuntu? Otherwise the private tangle is also not needed

Reply:
Yes we do